src/nvme/linux.c

```diff
@@ -443,23 +443,6 @@ char *nvme_get_path_attr(nvme_path_t p, const char *attr)
 	return nvme_get_attr(nvme_path_get_sysfs_dir(p), attr);
 }
 
-#ifndef CONFIG_OPENSSL
-int nvme_gen_dhchap_key(char *hostnqn, enum nvme_hmac_alg hmac,
```

Comment:
The dummy implementation if the library is not compiled with OpenSSL should stay IMO.

Reply:
oh I should revert that if fallback fails